presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -349,6 +349,9 @@ private boolean isColumnsEnumeratingTable(SchemaTableName schemaTableName)
         Optional<Set<String>> tables = filterString(constraint, TABLE_NAME_COLUMN_HANDLE);
         if (tables.isPresent()) {
             return prefixes.stream()
```

Comment:
can you assert  (`checkState` or `verify`) that you don't have fully qualified prefixes here?